src/devtools/client/webconsole/webconsole-ui.js

```diff
@@ -147,7 +110,7 @@ class WebConsoleUI {
 
     // Use a Custom Element to handle syntax highlighting to avoid
     // dealing with refs or innerHTML from React.
-    const win = this.window;
+    const win = window;
     win.customElements.define(
```

Comment:
I think we can just simplify this by calling `window` directly instead of setting it in a constant

Reply:
👍